acc_acc_modules/1_annotate_acc_modules.py

```diff
@@ -0,0 +1,373 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:core_acc] *
+#     language: python
+#     name: conda-env-core_acc-py
+# ---
+
+# # Add annotations
+#
+# This notebook takes the dataframe with information about module composition and their labels and adds additional annotations including:
+#
+# 1. Which gene is contained within the modules (both gene id and gene name)
+# 2. Baseline expression and expression in some context of interest
+# 3. How clustered the module is on the genome
+# 4. KEGG pathways that genes are found in
+# 5. GO pathways genes are found in
+# 6. Regulon/operon genes are found in
+#
+# All this information will help _P. aeruginosa_ experiments filter and determine which module might be interesting to explore.
+
+# +
+# %load_ext autoreload
+# %autoreload 2
+# %matplotlib inline
+import os
+import random
+import scipy
+import pandas as pd
+import numpy as np
+from itertools import product
+from core_acc_modules import paths
+
+random.seed(1)
+# -
+
+# User param
+method = "affinity"
+
+# +
+# Import gene memberships
+pao1_membership_filename = os.path.join(
+    paths.LOCAL_DATA_DIR, f"pao1_modules_{method}_acc.tsv"
+)
+pa14_membership_filename = os.path.join(
+    paths.LOCAL_DATA_DIR, f"pa14_modules_{method}_acc.tsv"
+)
+
+pao1_membership = pd.read_csv(pao1_membership_filename, sep="\t", index_col=0, header=0)
+pa14_membership = pd.read_csv(pa14_membership_filename, sep="\t", index_col=0, header=0)
+
+# +
+# Import gene metadata
+pao1_gene_annot_filename = paths.GENE_PAO1_ANNOT
+pa14_gene_annot_filename = paths.GENE_PA14_ANNOT
+
+pao1_gene_annot = pd.read_csv(pao1_gene_annot_filename, index_col=0, header=0)
+pa14_gene_annot = pd.read_csv(pa14_gene_annot_filename, index_col=0, header=0)
+# -
+
+# Import metadata of samples
+metadata_filename = paths.SAMPLE_METADATA
+
+pao1_gene_annot = pao1_gene_annot["Name"].to_frame("gene name")
```

Comment:
This is really nitpicky, but what is this code block doing? It's not clear from the functions themselves.

Reply:
`pao1_gene_annot["Name"]`: is selecting column "Name" from dataframe `pao1_annot`, which returns a pandas series, which I then convert to a dataframe by calling `to_frame()` which the column name "gene name". This will just allow me to do my merges that are in the notebook